CONFIG.md

```diff
@@ -99,6 +99,8 @@ ios:
     assetsFolder: Illustrations
     # Image name style: camelCase or snake_case
     nameStyle: camelCase
+    # An array of asset scales that should be downloaded. The valid values are 1, 2, 3.
+    scales: [1, 2, 3]
```

Comment:
This parameter should be marked as optional. Add info about default value.

`# [optional] An array of asset scales that should be downloaded. The valid values are 1, 2, 3. The default value are [1, 2, 3].`

Reply:
Done